sycl/doc/extensions/proposed/sycl_ext_intel_user_defined_reductions.asciidoc

```diff
@@ -0,0 +1,138 @@
+= sycl_ext_intel_user_defined_reductions
+:source-highlighter: coderay
+:coderay-linenums-mode: table
+
+// This section needs to be after the document title.
+:doctype: book
+:toc2:
+:toc: left
+:encoding: utf-8
+:lang: en
+
+:blank: pass:[ +]
+
+// Set the default source code type in this document to C++,
+// for syntax highlighting purposes.  This is needed because
+// docbook uses c++ and html5 uses cpp.
+:language: {basebackend@docbook:c++:cpp}
+
+== Notice
+
+Copyright (c) 2022 Intel Corporation.  All rights reserved.
+
+IMPORTANT: This specification is a draft.
```

Comment:
There's a new template for SYCL extensions: https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/template.asciidoc.

It changes more than just this section, but this is the first place I noticed you were using the old format.

Reply:
Thanks for the link! Yes, I simply copied the format from another existing extension. I've updated to the new one in a9dc59b3852d504a0fa8a30af8b852b4697e9c9a, but I have a few questions: do we still need to maintain revision history or it is now automatically done through feature test macro table? We don't anymore list extension contributors anywhere in the document, right?

Reply:
I'm pretty sure that revision history was replaced by the feature test macro.  The feature test macro only has to get updated if the API changes, and the changelog for everything else is in the commit history.

You don't need to list extension contributors.